cookiecutter-xblock/{{cookiecutter.repo_name}}/.github/workflows/ci.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       matrix:
         os: [ubuntu-20.04]
         python-version: ['3.8']
-        toxenv: [quality, docs, pii_check, django32, django40]
```

Comment:
Can you explain a little more about why you are removing these? What errors were you seeing? In general, we'd prefer to correct the errors in the templates rather than remove the checks.